_data/sites.json

```diff
@@ -8202,7 +8202,15 @@
             "framer.com"
         ]
     },
-
```

Comment:
Can you add this line break back, please?

Reply:
I will attempt to do so, check files formatting was complaining about my line spacing. I work in vscodium but i dont notice anything wrong

Reply:
It's because it has to be just a line break, without any spaces

Reply:
I think i got it, note to self: highlight the white dots in vscode / press f1 and type in trim trailing whitespace, and it should clear it up! (hopefully)